x/oracle/keeper/ballot.go

```diff
@@ -19,16 +17,12 @@ func (k Keeper) OrganizeBallotByDenom(
 	// collect aggregate votes
 	aggregateHandler := func(voterAddr sdk.ValAddress, vote types.AggregateExchangeRateVote) bool {
 		// organize ballot only for the active validators
-		claim, ok := validatorClaimMap[vote.Voter]
+		_, ok := validatorClaimMap[vote.Voter]
```

Comment:
Is this claim map only used to determine whether or not a validator is in the active set? seems like it's built here : 

```
		validatorClaimMap := make(map[string]types.Claim)
		for _, v := range k.StakingKeeper.GetBondedValidatorsByPower(ctx) {
			addr := v.GetOperator()
			validatorClaimMap[addr.String()] = types.NewClaim(0, addr)
		}
```

If so, we'll likely want to rename it to make that clear

Reply:
I put the power param back into the claim type so now its using the validator power